unittests/test_requirement_constraint_expression_evaluation.py

```diff
@@ -19,14 +19,14 @@ class TestRequirementConstraintEvaluation:
     """Test for the evaluation of the conditions tests (Mussfeldprüfung)"""
 
     # some valid nodes for easier referencing
-    _rc_1 = RequirementConstraint(condition_key="1", conditions_fulfilled=ConditionFulfilledValue.FULFILLED)
-    _rc_2 = RequirementConstraint(condition_key="2", conditions_fulfilled=ConditionFulfilledValue.UNFULFILLED)
-    _rc_3 = RequirementConstraint(condition_key="3", conditions_fulfilled=ConditionFulfilledValue.FULFILLED)
-    _rc_4 = RequirementConstraint(condition_key="4", conditions_fulfilled=ConditionFulfilledValue.UNFULFILLED)
-    _rc_101 = RequirementConstraint(condition_key="101", conditions_fulfilled=ConditionFulfilledValue.UNKNOWN)
-    _rc_102 = RequirementConstraint(condition_key="102", conditions_fulfilled=ConditionFulfilledValue.UNKNOWN)
+    _rc_1 = RequirementConstraint(condition_key="1", conditions_fulfilled=cfv.FULFILLED)
+    _rc_2 = RequirementConstraint(condition_key="2", conditions_fulfilled=cfv.UNFULFILLED)
+    _rc_3 = RequirementConstraint(condition_key="3", conditions_fulfilled=cfv.FULFILLED)
+    _rc_4 = RequirementConstraint(condition_key="4", conditions_fulfilled=cfv.UNFULFILLED)
+    _rc_101 = RequirementConstraint(condition_key="101", conditions_fulfilled=cfv.UNKNOWN)
+    _rc_102 = RequirementConstraint(condition_key="102", conditions_fulfilled=cfv.UNKNOWN)
     _h_501 = Hint(condition_key="501", hint="[501] Hinweis: Foo")
-    _h_502 = Hint(condition_key="502", hint="[502] Hinweis: Bar")
+    _h_502 = Hint(condition_key="502", hint="[502] Hinw" "eis: Bar")
```

Comment:
War das Absicht oder ist das ein Typo?
```suggestion
    _h_502 = Hint(condition_key="502", hint="[502] Hinweis: Bar")
```